src/App.tsx

```diff
@@ -38,34 +62,38 @@ type Actor = {
   name: string,
 }
 
-
-const divStyle = {
-  color: 'blue',
-  backgroundColor: 'blue',
-};
-
 interface IState {
   movies: Movie[];
   genres: Genre[];
   actors: Actor[];
-}
-
-const cardStyle = {
-  height: '100%',
-  display: 'flex',
-  flexDirection: 'column',
+  anchorEl: any,
+  formsDialog: any,
+  deleteDialog: boolean,
+  checkBoxDisplay: boolean,
+  checkBox: boolean,
+  postSuccessAlert: boolean,
+  postFailureAlert: boolean,
+  deleteAlert: boolean,
 }
 
 class App extends React.Component {
   state: IState = { 
     movies: [],
     genres: [],
     actors: [],
+    anchorEl: '',
+    formsDialog: false,
+    deleteDialog: false,
+    checkBoxDisplay: false,
+    checkBox: false,
+    postSuccessAlert: false,
+    postFailureAlert: false,
+    deleteAlert: false,
   };
 
   joinStr(list: string[]): string {
     if (list) {
-      return list.join(', ')
+    //  return list.join(', ')
```

Comment:
?

Reply:
I get this error when I don't comment this section out. We can talk about this maybe tomorrow:

TypeError: list.join is not a function

Reply:
It is breaking because some movies were created without a genres field, or with a malformed one.

try something like:

```
  joinStr(list: string[]): string {
    if (list && list instanceof Array) {
      return list.join(', ')
    }
    return ''
  }
```